### ISSUE TITLE
Undefined StorageClass

### ISSUE THREAD

**Issue by arnalisdaniel** [cviebrock/laravel-mangopay#2]
Hi,

I have followed the installation instructions, but, I had an "Undefined StorageClass" error on the ServiceProvider.php file at line 91.
I think it is because the condition is

``` php
!is_null($app['config']['mangopay']['StorageClass'])
```

instead of

``` php
isset($app['config']['mangopay']['StorageClass']) && !is_null($app['config']['mangopay']['StorageClass'])
```

Now it is working in my app.
What do you think about it ?

**Comment by cviebrock** [comment]
I agree. This would be as a result of PR #1 from @brayniverse.

Can you make a PR to fix this? And I'd use `array_get()`, rather than the `isset && ...` construct. 

Thanks!

**Comment by brayniverse** [comment]
Hey, sorry about that. I'll submit a PR as soon as I can. Unless somebody else does it first.

**Comment by cviebrock** [comment]
@DzKerr Let me know if this change fixes things for you (0.9.4 release)
